app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeController.scala

```diff
@@ -163,7 +163,7 @@ class RepresenteeController @Inject() (
             journey.fold(_.ggCredId, _.ggCredId)
           ) { _ =>
             val backLink = answers.fold(
-              _ => routes.RepresenteeController.isFirstReturn(),
+              _ => triageRoutes.CommonTriageQuestionsController.furtherReturnHelp(),
```

Comment:
if they said it was the first return, the back link should go to the `isFirstReturn` page